extras/rya.console/src/test/java/mvm/rya/shell/RyaAdminCommandsTest.java

```diff
@@ -123,6 +124,9 @@ public void deletePCJ() throws InstanceDoesNotExistException, RyaClientException
 
     @Test
     public void getInstanceDetails() throws InstanceDoesNotExistException, RyaClientException {
+        // This test is failed if the default timezone was not EST, so now it's fixed at EST.
+        // If you get assert mismatch of EST!=EDT, try the deprecated getTimeZone("EST") instead.
+        TimeZone.setDefault(TimeZone.getTimeZone("America/New_York"));
```

Comment:
You made a good point in in the JIRA ticket... I think everything worked for me because the order of the test execution.  Can you move this to a static class initializer or a BeforeClass method (so that the order of the test within this file doesn't break things)?